src/components/CustomTextInput.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { ReactElement, useState } from 'react'
+import { TextInputProps } from 'react-native'
+import { widthPercentageToDP as wp } from 'react-native-responsive-screen'
+import styled, { useTheme } from 'styled-components/native'
+
+import { CloseIcon } from '../../assets/images'
+import theme from '../constants/theme'
+import { ContentError } from './text/Content'
+
+const LINE_HEIGHT = 30
+const MIN_HEIGHT = 60
+
+const StyledTextInput = styled.TextInput`
```

Comment:
Please use the lunes text components. You can use `ContentTextLight` here and remove font-size, font-weight and font-family then.

Reply:
this is a styled `textInput` not `text`

Reply:
sorry, you are right. Just fine as it is.